docs/learn/learn-system-chains.md

```diff
@@ -104,10 +123,19 @@ Polkadot network.
 
 Some of these collectives are the
 [Polkadot Alliance](https://polkadot.polkassembly.io/referendum/94) and the Polkadot Technical
-Fellowship. These on-chain collectives will play important roles in the future of network
-stewardship and decentralized governance.
+[Fellowship](./learn-opengov.md#fellowship). These on-chain collectives will play important roles in
+the future of network stewardship and decentralized governance.
+
+Networks themselves can act as collectives and express their legislative voices as single opinions
+within other networks. This is achieved with the assistance from a [bridge hub](#bridge-hubs).
+
+### Bridge Hubs
 
-### Bridge Hub
+Before Polkadot and Kusama supported their first parachains, the only way to design a bridge was to
+put the logic onto the Relay Chain itself. Since both networks now support parachains, it makes
+sense to have a parachain on each network dedicated to bridges. This is because of the execution
+isolation provided by parachains. The activity on a parachain does not affect the Relay Chain or
+other parachains, so a bridge hub can support bridges to many other consensus systems.
```

Comment:
Removing the ambiguous statement. The activity on parachain does affect the relaychain and other parachains (through XCM). Probably the sentence can benefit from rephrasing or be removed completely. 

```suggestion
isolation provided by parachains.
```